src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -55,6 +56,7 @@
 #include "runtime/javaCalls.hpp"
 #include "runtime/jniHandles.hpp"
 #include "runtime/os.hpp"
+#include "runtime/threads.hpp"
```

Comment:
Is the `#include` lines a leftover from the initial version? Do we still need them?

Reply:
Yes, undoing that, thanks!